docs/topics/kotlin-language-features-and-proposals.md

```diff
@@ -0,0 +1,384 @@
+[//]: # (title: Kotlin language features and proposals)
+[//]: # (description: Learn about Kotlin features lifecycle. 
+The page contains the full list of Kotlin language features and design proposals.)
+
+JetBrains evolves the Kotlin language according to the [Kotlin language evolution principles](kotlin-evolution-principles.md),
+driven by the pragmatic design. 
+You can find the list of language feature proposals and their statuses.
+
+## Kotlin language features lifecycle
+
+The Kotlin language feature can have one of the following statuses:
+
+* **Exploration and design**. We are considering the introduction of a new feature to the language.
+  This involves discussing how it would integrate with existing features, gathering use cases, 
+  and assessing its potential impact.
+  We need feedback from users on the problems this feature would solve and the use cases it addresses.
+  Estimating how often these use cases and problems occur would also be beneficial.
```

Comment:
Do you mean that it would be beneficial to receive estimates from users how often such use cases and problems occur? It's not clear to me 🤔

Reply:
Hmm, let me figure out how to rephrase that.